builtin/logical/mongodb/path_config_lease.go

```diff
@@ -0,0 +1,107 @@
+package mongodb
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathConfigLease(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "config/lease",
+		Fields: map[string]*framework.FieldSchema{
+			"ttl": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "Default ttl for credentials.",
+			},
+
+			"ttl_max": &framework.FieldSchema{
```

Comment:
`mssql` using it was an unfortunate oversight during review. `aws`, `mysql`, and `postgresql` have never been updated to be more current.